packages/world/src/systems/BuildSystem.sol

```diff
@@ -284,7 +288,5 @@ library BuildLib {
     // if (terrainType == ObjectTypes.Water && !buildType.isWaterloggable()) {
     //   EntityFluidLevel._deleteRecord(terrain);
     // }
-
-    EntityObjectType._set(terrain, buildType);
```

Comment:
Why dont we need this anymore?

Reply:
I moved it back to the addBlocks function